spec/libraries/date_time_helpers.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Extensions::DateTimeHelpers do
+  describe '.min' do
+    it 'is a valid time in database' do
+      expect{User.where { created_at > Time.min }}.not_to raise_error
+    end
+  end
+
+  describe '.max' do
+    it 'is a valid time in database' do
+      expect{User.where { created_at < Time.max }}.not_to raise_error
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.